src/apps/content-editor/src/app/views/ItemEdit/Meta/ItemSettings/settings/ItemRoute.js

```diff
@@ -37,6 +37,7 @@ export const ItemRoute = connect(state => {
 
         return request(`${CONFIG.API_INSTANCE}/search/items?q=${fullPath}`)
```

Comment:
If possible we should opt for the action creator searchItems: https://github.com/zesty-io/manager-ui/blob/master/src/shell/store/content.js#L265

Reply:
I don't think I can use the SearchItem reducer as ItemRoute is looking for direct path. Please let me know if I am incorrect on how to consume the SearchItem reducer?